packages/fast-colors/src/rgb.spec.ts

```diff
@@ -0,0 +1,186 @@
+        expect(hexToRgb("#ffff")).toBe("rgba(255, 255, 255, 1)");
+    });
+    test("Should return rgba(0, 0, 0, 0)", (): void => {
+        expect(hexToRgb("#0000")).toBe("rgba(0, 0, 0, 0)");
+    });
+    test("Should return rgba(255, 0, 0, 0.8)", (): void => {
+        expect(hexToRgb("#F00C")).toBe("rgba(255, 0, 0, 0.8)");
+    });
+    test("Should return rgba(221, 204, 153, 1)", (): void => {
+        expect(hexToRgb("#DC9F")).toBe("rgba(221, 204, 153, 1)");
+    });
+    test("Should return rgba(17, 51, 238, 0.6)", (): void => {
+        expect(hexToRgb("#13E9")).toBe("rgba(17, 51, 238, 0.6)");
+    });
+    test("Should return rgba(255, 255, 51, 0.06666666666666667)", (): void => {
+        expect(hexToRgb("#fF31")).toBe("rgba(255, 255, 51, 0.06666666666666667)");
+    });
+    test("Should return rgba(51, 153, 136, 0.4)", (): void => {
+        expect(hexToRgb("#3986")).toBe("rgba(51, 153, 136, 0.4)");
+    });
+    test("Should return rgba(170, 170, 170, 0.6666666666666666)", (): void => {
+        expect(hexToRgb("#AaAa")).toBe("rgba(170, 170, 170, 0.6666666666666666)");
+    });
+    test("Should return rgba(17, 187, 119, 0.8666666666666667)", (): void => {
+        expect(hexToRgb("#1B7d")).toBe("rgba(17, 187, 119, 0.8666666666666667)");
+    });
+    test("Should return rgba(238, 153, 119, 0.2)", (): void => {
+        expect(hexToRgb("#e973")).toBe("rgba(238, 153, 119, 0.2)");
+    });
+});
+
+describe("Valid 6 digit hexits", (): void => {
+    test("Should return rgb(255, 255, 255)", (): void => {
+        expect(hexToRgb("#ffffff")).toBe("rgb(255, 255, 255)");
+    });
+    test("Should return rgb(0, 0, 0)", (): void => {
+        expect(hexToRgb("#000000")).toBe("rgb(0, 0, 0)");
+    });
+    test("Should return rgb(241, 242, 253)", (): void => {
+        expect(hexToRgb("#f1f2fd")).toBe("rgb(241, 242, 253)");
+    });
+    test("Should return rgb(252, 233, 130)", (): void => {
+        expect(hexToRgb("#fce982")).toBe("rgb(252, 233, 130)");
+    });
+    test("Should return rgb(205, 233, 143)", (): void => {
+        expect(hexToRgb("#cdE98F")).toBe("rgb(205, 233, 143)");
+    });
+    test("Should return rgb(168, 76, 93)", (): void => {
+        expect(hexToRgb("#A84C5D")).toBe("rgb(168, 76, 93)");
+    });
+    test("Should return rgb(25, 216, 55)", (): void => {
+        expect(hexToRgb("#19D837")).toBe("rgb(25, 216, 55)");
+    });
+    test("Should return rgb(132, 147, 224)", (): void => {
+        expect(hexToRgb("#8493E0")).toBe("rgb(132, 147, 224)");
+    });
+    test("Should return rgb(212, 124, 138)", (): void => {
+        expect(hexToRgb("#D47C8A")).toBe("rgb(212, 124, 138)");
+    });
+    test("Should return rgb(188, 218, 158)", (): void => {
+        expect(hexToRgb("#BCDA9E")).toBe("rgb(188, 218, 158)");
+    });
+});
+
+describe("Valid 8 digit hexits", (): void => {
+    test("Should return rgba(255, 255, 255, 1)", (): void => {
+        expect(hexToRgb("#FFFFFFFF")).toBe("rgba(255, 255, 255, 1)");
```

Comment:
I think you could combine these and try more of a grouping strategy, if it's wrong we will see the expected value is not being met so it's not super necessary to put the expected value in the test stdout string, you could group some like minded tests with "should convert when all alphabetical characters in provided hex are uppercase", "should convert when alphabetical characters in provided hex are a mix of upper and lower case", "should convert if provided hex is missing a '#' symbol"